cli/cmd/testdata/install_output.golden

```diff
@@ -866,7 +866,7 @@ rules:
   verbs: ["create"]
 - apiGroups: [""]
   resources: ["configmaps"]
-  resourceNames: [CertificateBundleName]
+  resourceNames: [TLSTrustAnchorConfigMapName]
```

Comment:
I notice that all of the other fields are lists of quoted strings. Is this valid YAML?

Reply:
IDK why that isn't quoted but this PR doesn't change the quotedness and I didn't want to change the quotedness since it seems out of scope.

Reply:
Another way to phrase this: have we actually loaded the injected output into kubernetes yet (i.e. with `--tls=optional`)? I suspect that this may not actually have been run through a yaml parser yet...

Reply:
This is valid YAML -- strings don't need quotes unless they have special characters or could be mistaken for another type, such as int. But I agree that for consistency it would be good to quote it. That said, I've tested the RBAC configs as written and they work.